CHANGELOG.next.asciidoc

```diff
@@ -121,10 +122,6 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 *Metricbeat*
 
-- Extend documentation about `orchestrator.cluster` fields {pull}30518[30518]
-- Enhance Oracle Module: Change tablespace metricset collection period {issue}30948[30948] {pull}31259[#31259]
-- Add orchestrator cluster ECS fields in kubernetes events {pull}31341[31341]
-- Add new Kubernetes module dashboards {pull}31341[31341]
```

Comment:
It looks like these three are all for 8.3. Do we want to delete them from this _next_ changelog?

Reply:
The thing is that they are in the 8.2.1 changelog, probably a bad backport, I'll take a look

Reply:
Fixed in a later commit for `CHANGELOG.next`, the changelog will be fixed in a follow-up PR

Reply:
Sounds good!